.github/workflows/ci-jest-coverage.yml

```diff
@@ -0,0 +1,15 @@
+name: 'coverage'
+on:
+    pull_request:
+        branches:
+            - master
+            - main
+jobs:
+    coverage:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v3
+            - uses: ArtiomTr/jest-coverage-report-action@v2
```

Comment:
Thoughts on folding this into the existing test workflow so it doesn't have to rerun the tests? 

> To bypass running unit tests, you can pass the filepath to the current report.json

https://github.com/ArtiomTr/jest-coverage-report-action?tab=readme-ov-file#use-existing-test-reports

Reply:
Yeah, I think ultimately we'd want to have it run through only once.

Reply:
OK, I think I've got it incorporated into the existing workflow such that the only extra work it is doing is the necessary run of tests for the `main` branch (for comparison).